2-WebApp-graph-user/2-1-Call-MSGraph/WebApp-OpenIDConnect-DotNet-graph.csproj

```diff
@@ -1,15 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
     <UserSecretsId>aspnet-WebApp_OpenIDConnect_DotNet_graph-0701D544-D6FC-4E5A-84E4-B8DD052BA93F</UserSecretsId>
     <WebProject_DirectoryAccessLevelKey>0</WebProject_DirectoryAccessLevelKey>
     <RootNamespace>CallMSGraph</RootNamespace>
     <AssemblyName>CallMSGraph</AssemblyName>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="3.1.14" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="5.0.10" />
```

Comment:
I don't think that this package is needed any longer. Is it?

Reply:
I didn't want to remove the package because I don't know what is it for. I will remove it and we will review it with Kalyan tomorrow.

Reply:
@jmprieur ,  I can change all the samples to 5.0, if you don't object to it.
I've sent an email earlier today to you and to @jennyf19 about build agent fails on missing 5.0 references. Before going ahead I'd like to get a clean build.

Reply:
I don't object. We'll need to update the README.md files too.
I've replied to your email, @aremo-ms: I've updated the build.

Reply:
Oh, that's great. Thanks
We will review the PR tomorrow and I will go ahead with update both for 5.0 and for readme(s)